helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/MessageProxyService.java

```diff
@@ -122,25 +109,24 @@ private void createAgentProxy(Ticket ticket, HelpDeskAiSessionContext aiSessionC
 
     helpDeskAi.startConversation(aiSessionContext.getAiSessionKey(), aiConversation);
 
-    aiSessionContext.setProxyConversation(aiConversation);
+    aiSessionContext.setIdleTimerManager(idleTimerManager);
 
     MessageProxy messageProxy = new MessageProxy();
-    messageProxy.setAgentProxyTimer(new ProxyIdleTimer(idleTicketConfig.getTimeout(),
+    messageProxy.setProxyIdleTimer(new ProxyIdleTimer(idleTicketConfig.getTimeout(),
```

Comment:
Why do you still need MessageProxy class? It seems unnecessary.

Reply:
ok